plugin.yml

```diff
@@ -12,8 +12,10 @@ prefix: "[BedWars]"
 permissions:
   bedwars.command:
     description: Base perm for bedwars commands
-    default: op
+    default: true
```

Comment:
nope

Reply:
Please explain why? When OP no one can run the other commands without having permission for bedwars.command. This needlessly requires you to negate permissions for everything you don't want a user to run even though some of those commands aren't OP required

Reply:
Just realized - someone reported that issue once, it is correct to use "true". #blameshoghi for the bad permission design in PMMP